web-frame/src/app/app.module.ts

```diff
@@ -28,6 +29,14 @@ import { AppComponent } from './app.component';
     CoreModule.forRoot(),
     SecurityModule.forRoot(),
   ],
+  providers: [
+    APP_ERROR_HANDLER_PROVIDER,
```

Comment:
I am having a hard time to follow what is the convention for naming here, if we should use upper case or camelcase. Why did you decide to make it upper case?

Reply:
We use SNAKE_CASE for all constants and this constant references a `Provider` that provides a custom error handler. And I kept that provider next to the actual error handler to having many imports in the `app.module.ts`.